tests/integration/build_invoke/test_build_invoke_dotnetcore2_1.py

```diff
@@ -0,0 +1,22 @@
+import pytest
```

Comment:
Why dotnet and golang build invoke tests are in separate file?

Reply:
Update comment:

```
sam build does not support to build go 1.x templates using container, 
here we put them in a separate file and use a dedicate codebuild project with go 1.x runtime to build them.
```